misk-hibernate/src/main/kotlin/misk/jdbc/ScaleSafetyChecks.kt

```diff
@@ -0,0 +1,5 @@
+package misk.jdbc
+
+interface ScaleSafetyChecks {
+  fun <T> disable(body: () -> T): T
```

Comment:
I don't think this API is obvious... Is it just “runs body if checks are disabled" ?

Reply:
Nope, its “runs body with checks disabled” !

I think this probably wants to be a offered via an on Session? That's already handy when you're gonna wanna disable checks right?

Reply:
Done! I like it! My main gripe is that now I have to provide a `Transacter::currentSession` method which I'm not too happy about. Do you have other suggestions?

Reply:
Yes! The API goes on Session but it manipulates a property of the transacter